src/config.ts

```diff
@@ -80,6 +79,11 @@ export default function getConfig(): Config {
  * @param deploymentID The deployment ID in the config.
  */
 export function getHost(deploymentID: string): string {
+  const re = /^http.*:\/\/[a-zA-Z0-9\-._]+:[0-9]+/;
```

Comment:
Do you have a source for the this regex? Want to make sure we're catching all the corner cases.

Reply:
I just tested this from https://regex101.com/ with:

- http://localhost:8080
- https://localhost:8081